app/components/post_textbox/post_textbox_base.js

```diff
@@ -685,49 +685,49 @@ export default class PostTextBoxBase extends PureComponent {
         });
     };
 
-    showPasteImageErrorDialog = () => {
+    showPasteFilesErrorDialog = () => {
         const {formatMessage} = this.context.intl;
         Alert.alert(
             formatMessage({
-                id: 'mobile.image_paste.error_title',
-                defaultMessage: 'Paste Image failed',
+                id: 'mobile.files_paste.error_title',
+                defaultMessage: 'Paste files failed',
```

Comment:
Total nitpicking, but this might sound a bit better
```suggestion
                defaultMessage: 'Paste failed',
```

Reply:
sure I can change it, and also I need to modify en.json that was not included in this PR